packages/insomnia-app/app/common/har.ts

```diff
@@ -172,6 +173,33 @@ export interface Har {
 export interface ExportRequest {
   requestId: string;
   environmentId: string | null;
+  responseId?: string;
+}
+
+export async function exportHarCurrentRequest(request: Request, response: Response): Promise<Har> {
+  const ancestors = await database.withAncestors(request, [
+    models.workspace.type,
+    models.requestGroup.type,
+  ]);
+  const workspace = ancestors.find(ancestor => ancestor.type === models.workspace.type);
```

Comment:
We can use `isWorkspace` here

```suggestion
  const workspace = ancestors.find(isWorkspace);
```

Reply:
good one 7747354e58cfc50131dbf581228aaf4eac945a89